### DISCUSSION TITLE
Image tags and albums?

### DISCUSSION THREAD

**Discussion by abrslam** [photoview/photoview#208] [Ideas]
The two features I would be most interested in are the ability to tag photos and/or sort into collections or albums (without changing the actual location of images on the server).

**Comment by viktorstrate** [comment]
Hi @abrslam I would like to hear your thoughts on how this might work in practice, what changes to the UI should be made.

Also I guess it would be nice if it played together with #145

**Reply by abrslam** [reply]
My thinking was to have tags listed under the rest of the photo metadata (camera, focal length, flash etc).  With links for each of the tags (clicking on a link will bring you to a list of all the photos with that tag), as well as the option to add more tags.  

For albums my thinking was another box similar to "download options" listing the album a photo belongs to, as well as the option to "add to album".

Also "show only favourites" is a good option, but stuck at the top is a bit of an issue.  It's trouble to turn it on and off when you've scrolled very far down the timeline.  Perhaps a toggle on the left panel?  Or perhaps a drop down box with favourites as well as albums?

**Reply by viktorstrate** [reply]
I like your thoughts on how tags might work, and I think that could quite easily be combined with #145.

As for moving images to other albums I think that's outside of the aim of the project.
I think one of the strengths of Photoview is that it always reflects the structure of the filesystem, and this would break that idea.

**Comment by Stankye** [comment]
Image tag support for [DarkTable](https://www.darktable.org/) sidecar files (just some xml files) would be amazing. Being able to search said tags would also be amazing but would probably require some sort of database. I enjoy how https://github.com/simon987/sist2 handles tag searching and display.

Not on discussion topic but the introduction to tag searching would give the structure needed to add 'star' ratings and other such goodies.
